mysql/README.md

```diff
@@ -4,42 +4,163 @@
+| `performance_schema_max_digest_length` | `4096` | Must match `max_digest_length`. |
+| `performance_schema_max_sql_text_length` | `4096` | Must match `max_digest_length`. |
+
+**Note**: For Amazon RDS MySQL, there is no way to configure the `events-statements_*` consumers in the [DB Parameter Group][42] so they must enabled dynamically at runtime. See [Runtime Setup Consumers](#runtime-setup-consumers).
+
+<!-- xxz tab xxx -->
+<!-- xxx tab "Amazon RDS Aurora MySQL" xxx -->
+
+Configure the following in the [DB Parameter Group][43]:
+
+| Parameter | Value | Description |
+| --- | --- | --- |
+| `performance_schema` | `ON` | Required. Enables the [Performance Schema][41]. |
+| <code style="word-break:break-all;">performance_schema_consumer_events_statements_current</code> | `ON` | Required. Enables monitoring of currently running queries. |
+| <code style="word-break:break-all;">performance_schema_consumer_events_statements_history</code> | `ON` | Optional. Enables tracking recent query history per thread. If enabled it increases the likelihood of capturing execution details from infrequent queries. |
+| <code style="word-break:break-all;">performance_schema_consumer_events_statements_history_long</code> | `ON` | Optional. Enables tracking of a larger number of recent queries across all threads. If enabled it increases the likelihood of capturing execution details from infrequent queries. |
+| `max_digest_length` | `4096` | Required for collection of larger queries. Increases the size of SQL digest text in `events_statements_*` tables. If left at the default value then queries longer than `1024` characters will not be collected. |
+| <code style="word-break:break-all;">`performance_schema_max_digest_length`</code> | `4096` | Must match `max_digest_length`. |
+| <code style="word-break:break-all;">`performance_schema_max_sql_text_length`</code> | `4096` | Must match `max_digest_length`. |
+
+<!-- xxz tab xxx -->
+<!-- xxz tabs xxx -->
+
+### Agent Database Access
+
+The Datadog Agent requires read-only access to the database in order to collect statistics and queries.
+
+The following instructions grant the agent permission to login from any host using `datadog@'%'`. This is required for managed databases like `Amazon RDS` or `Google CloudSQL` as it's not possible to install the datadog agent directly on the host. For self-hosted databases the datadog user can be restricted to be allowed to login only from localhost by using `datadog@'localhost'`. See the [MySQL documentation][5] for more info.
+
+<!-- xxx tabs xxx -->
+<!-- xxx tab "MySQL ≥ 8.0" xxx -->
+
+```SQL
+CREATE USER datadog@'%' IDENTIFIED WITH mysql_native_password by '<UNIQUEPASSWORD>';
+ALTER USER datadog@'%' WITH MAX_USER_CONNECTIONS 5;
+GRANT REPLICATION CLIENT ON *.* TO datadog@'%'
```

Comment:
Missing a semicolon at the end: `GRANT REPLICATION CLIENT ON *.* TO datadog@'%';`

Reply:
Fixed